docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,428 @@
+
+#. Run the following command to create a data pipeline job named ``simple_agg``:
+
+   ::
+
+      avn service flink job create demo-flink simple_agg        \
+          --table-ids CPU_IN_ID CPU_OUT_AGG_ID                  \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN`` and ``CPU_OUT_AGG`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 35-49
+      :language: sql
+
+
+Create a Flink SQL job using PostgreSQL thresholds
+--------------------------------------------------
+
+This setup uses host-specific thresholds that are stored in PostgreSQL as a basis for determining instances of high CPU load.
+
+.. mermaid::
+
+    graph LR;
+
+        id1(Kafka source)-- metrics stream -->id3(Flink job);
+		id2(PosgreSQL source)-- host-specific thresholds -->id3;
+        id3-- host with high CPU -->id4(Kafka sink);
+
+
+1. In the Aiven CLI, run the following command to connect to the ``demo-postgresql`` service:
+   
+   ::
+	  
+      avn service cli demo-postgresql --project PROJECT_NAME
+   
+#. Enter the following commands to set up the PostgreSQL table containing the threshold values:
+   
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 52-53
+      :language: sql
+
+#. Enter the following command to check that the threshold values are created:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 54
+      :language: sql
+
+   The output shows you the content of the table:
+
+   ::
+
+      hostname | allowed_top
+      ---------+------------
+      doc      |     20
+      grumpy   |     30
+      sleepy   |     40
+      bashful  |     60
+      happy    |     70
+      sneezy   |     80
+      dopey    |     90
+
+#. Run the following command to create a PostgreSQL table named ``SOURCE_THRESHOLDS``:
+
+   ::
+
+      avn service flink table create demo-flink POSTGRESQL_INTEGRATION_ID  \
+          --table-name SOURCE_THRESHOLDS                                   \
+          --jdbc-table cpu_thresholds                                      \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``POSTGRESQL_INTEGRATION_ID`` with the ID for your ``demo-postgresql`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 57-59
+      :language: sql
+
+#. Run the following command to create a Kafka table named ``CPU_OUT_FILTER_PG``:
+
+   ::
+
+      avn service flink table create demo-flink KAFKA_INTEGRATION_ID  \
+          --table-name CPU_OUT_FILTER_PG                              \
+          --kafka-topic cpu_load_stats_real_filter_pg                 \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``KAFKA_INTEGRATION_ID`` with the ID for your ``demo-kafka`` service integration and replace ``TABLE_SQL`` with the following:
```

Comment:
Again, placeholder values to be included before the place they are described. I'd also like an explanation of the SQL being used and what it does, if possible.

Include the code samples using labels rather than line numbers